legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -172,3 +174,35 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
     return msg
+
+def validate_founding_date(filing_json: dict, filing_type: str) -> list:
```

Comment:
do this validation inside validate_business_in_colin and just compare with the founding date received from colin

Reply:
Got it, I will update it

Reply:
Ohhh, sorry Megan, I misled you! This validation isn't to ensure that the date is a valid date, but rather to ensure that the date provided by the UI matches the date from COLIN (ie, when the expro business was searched and the fields in the UI were pre-filled).

I guess it's possible that COLIN doesn't have the date but the user provides it. In that case, should we just accept it or validate that, say, it's in the past?

Reply:
is it ever possible to have a expro (or any other business) without founding date?

Reply:
this is founding date of expro (not the incorporation date in foreign jurisdiction)

Reply:
I think there are 2 dates:
- the date the expro was registered in BC
- the date of incorporation in the previous jurisdiction

It is the second date that is prompted in the UI that we are talking about. Does it need validation? (The UI pre-fills from COLIN data but the user can change it.)

![image](https://github.com/user-attachments/assets/da2b602a-26fb-4a78-9343-ba5c007fbd1a)

Reply:
validation of incorporation date in foreign jurisdiction should be checked in `_validate_foreign_jurisdiction`, which comes inside `filing_json['filing'][filing_type]['foreignJurisdiction']['incorporationDate']`

Reply:
I can update a validation for ['foreignJurisdiction']['incorporationDate']. May I know what should be validated? (Valid format, before today, anything else?)

Reply:
> I can update a validation for ['foreignJurisdiction']['incorporationDate']. May I know what should be validated? (Valid format, before today, anything else?)

Sounds good to me.